openapi.yml

```diff
@@ -689,6 +689,87 @@ components:
           type: string
           format: date-time
 
+    UpdateCircuit: # Can be referenced via '#/components/schemas/UpdateCircuit'
+      type: object
+      additionalProperties: false
+      properties:
+        uni_a:
+          $ref: '#/components/schemas/Endpoint'
+        uni_z:
+          $ref: '#/components/schemas/Endpoint'
+        name:
+          type: string
+        request_time:
```

Comment:
`flow_removed_at`, `execution_rounds` shouldn't be exposed to be updated via API. 

`request_time` seems to be redundant with `creation_time`, but it's something to discuss in another issue. `request_time` in the docstring in the code was for:

```
        # datetime of user request for a EVC (or datetime when object was
        # created)
```

Regardless, I don't think `request_time` should be externally set either, the NApp should set this value without relying on a value from the API.

Reply:
Removed from api document